docs/configuration.md

```diff
@@ -1398,6 +1398,15 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.scheduler.listenerbus.eventqueue.size</code></td>
```

Comment:
If you're only documenting this in master then please use `spark.scheduler.listenerbus.eventqueue.capacity` instead (see definition in code and git blame for explanation).